Framework/DataHandling/src/LoadInstrument.cpp

```diff
@@ -201,16 +209,39 @@ void LoadInstrument::exec() {
     if (!m_filename.empty())
       runLoadParameterFile();
   }
+}
 
-  // Set the monitors output property
-  setProperty("MonitorList", instrument->getMonitors());
+void LoadInstrument::NexusInstrumentLoader() {
+  checkAndRetrieveInstrumentFilename(FileType::Nexus);
```

Comment:
At the moment we couldn't get to a point where we were looking for a nexus file based on a instrument name. Is that correct?